docs/tutorials/03_shaders/viz_sdf_cylinder.py

```diff
@@ -0,0 +1,296 @@
+"""
+===============================================================================
+Make a Cylinder using polygons vs SDF
```

Comment:
SDF ?  Self Defense Force ? 😄 ,  The acronym definition is never shown entirely. Also, one short sentence to explain it would be great.